bitmap-ittybitty/lib/handler.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
Nice work!

Since functions are hoisted, I would probably group together all your exports stuff either at the top or the bottom, not spread out through the file.